backend/src/main/kotlin/ch/learntrack/backend/backoffice/user/UserService.kt

```diff
@@ -30,11 +30,13 @@ public class UserService(
     public fun getAllAdminUsers(): List<UserDto> = userDao.fetchAllAdminUsers().map(::mapToDto)
 
     public fun createAdminUser(createUserDto: CreateUserDto) {
-        if (!isEmailValid(createUserDto.email)) {
+        val emailLowerCase = createUserDto.email.trim().lowercase()
+
+        if (!isEmailValid(emailLowerCase)) {
             throw LearnTrackConflictException("Invalid email")
         }
 
-        if (isEmailExisting(createUserDto.email)) {
+        if (isEmailExisting(emailLowerCase)) {
             throw LearnTrackConflictException("Email already exists")
```

Comment:
i just saw this yet 😂. could you print the email in the Exception so if one occures  we understand why. you can use a template string like this.
```suggestion
            throw LearnTrackConflictException("Email ${emailLowerCase} already exists")
```

Reply:
also for the exception above too, the one with the invalid email